examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java

```diff
@@ -467,7 +467,6 @@ public void processElement(ProcessContext c) {
 
   public static void runAutocompletePipeline(Options options) throws IOException {
 
-    options.setBigQuerySchema(FormatForBigquery.getSchema());
```

Comment:
Why are these being removed?

Reply:
The property bigQuerySchema fails to serialize and deserialize for portable runners as part of PipelineOptions. 
Stacktrace snippet
`Failed to serialize PipelineOptions
java.lang.IllegalArgumentException: Failed to serialize PipelineOptions
	at org.apache.beam.runners.core.construction.SerializablePipelineOptions.serializeToJson(SerializablePipelineOptions.java:70)`
`Caused by: com.fasterxml.jackson.databind.JsonMappingException: Unexpected IOException (of type java.io.IOException): Failed to serialize and deserialize property 'bigQuerySchema' with value 'GenericData{classInfo=[fields], `

Reply:
For these examples, the schema is added directly to the transform https://github.com/apache/beam/blob/46155767ecf0694d86257e9c3a8b5fd5bbc0afbb/examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java#L515 using `FormatForBigquery.getSchema()`, not sure why was added to options if it wasn't used from there.